pkg/prometheus/promcfg_test.go

```diff
@@ -8830,10 +8882,144 @@ scrape_configs:
 				nil,
 				nil,
 				nil,
+				nil,
 			)
 			require.NoError(t, err)
 			require.Equal(t, tc.expectedCfg, string(cfg))
 		})
 
 	}
 }
+
+func TestTracingConfig(t *testing.T) {
+	boolFalse := false
+	testCases := []struct {
+		tracingConfig  *monitoringv1.PrometheusTracingConfig
+		name           string
+		expectedConfig string
+		shouldSucceed  bool
```

Comment:
nit: just to be consistent with other similar code in this file
```suggestion
		expectedErr  bool
```